data/sql/updates/pending_db_world/rev_1609431870979648000.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609431870979648000');
+UPDATE `gameobject_template` SET `Data2` = 0, `AIName` = 'SmartGameObjectAI' WHERE `entry` = 181964;
+DELETE FROM `event_scripts` WHERE `id` = 11027;
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (181964, 17715, 1771500);
```

Comment:
you need to specify the `source_type` as well when using `DELETE` on `smart_scripts`

Reply:
done